### ISSUE TITLE
Scheduled Backups

### ISSUE THREAD

**Issue by Lagicrus** [AdguardTeam/AdGuardHome#6822]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

When reinstalling AdGuard via the provided script, it overwrote my .yaml file containing months of configurations and settings I had applied with seemingly no recourse to reverse that action or find a backup to replace it as I had expected the reinstall process to either not wipe the config, or use it in the new build.

### Proposed solution

Have a setting where the AdGuard yaml file which holds all the data, gets backed up on a set schedule such as every week

**Comment by whyisthisbroken** [comment]
I don't know, but once you've set it up, you copy the Yaml and save it somewhere as a backup.
Once AGH is running, there is no need to change the settings every week, so a weekly backup would be necessary. This is wasted development effort to integrate a backup. 

And why are you reinstalling AGH? Shouldn't updating via GUI be enough?

**Comment by Lagicrus** [comment]
@whyisthisbroken  I agree that solutions like rclone exist. Still, I see no harm in an inbuilt solution, which could be set to run from the second you install it for the first time instead of potentially forgetting until it's too late.

For why I reinstalled it, see https://github.com/AdguardTeam/AdGuardHome/issues/6811, but the short version is that after updating via the GUI, all UI/CSS broke, making it unusable.
